viewer/packages/360-images/src/entity/Image360Entity.ts

```diff
@@ -37,111 +24,89 @@ export class Image360Entity implements Image360 {
     return this._image360Icon;
   }
 
-  /**
-   * The object containing the unit cube with the 360 images.
-   * @returns Image360Visualization
-   */
-  get image360Visualization(): Image360VisualizationBox {
-    return this._image360VisualzationBox;
-  }
-
   constructor(
-    image360Metadata: Image360Descriptor,
+    image360Metadata: Historical360ImageSet,
     sceneHandler: SceneHandler,
     imageProvider: Image360FileProvider,
     transform: THREE.Matrix4,
-    icon: Image360Icon
+    icon: Image360Icon,
+    reloadImage: (entity: Image360Entity, revision: Image360RevisionEntity) => Promise<void>
   ) {
-    this._imageProvider = imageProvider;
-    this._image360Metadata = image360Metadata;
-
-    this._transform = transform;
     this._image360Icon = icon;
-    this._image360VisualzationBox = new Image360VisualizationBox(this._transform, sceneHandler);
-    this._image360VisualzationBox.visible = false;
-    this._getFullResolutionTextures = undefined;
+    this._reloadImage = reloadImage;
+
+    this._revisions = image360Metadata.imageRevisions.map(descriptor => {
+      return new Image360RevisionEntity(imageProvider, descriptor, sceneHandler, transform);
+    });
```

Comment:
Note that simply commiting this suggestion will probably have lint error.

```suggestion
    this._revisions = image360Metadata.imageRevisions.map(descriptor => new Image360RevisionEntity(imageProvider, descriptor, sceneHandler, transform));
```